vnext/Scripts/Microsoft.ReactNative.nuspec

```diff
@@ -20,6 +20,7 @@
     Uncomment once we are producing doc xml file
     <file src="$nugetroot$\x64\Release\Microsoft.ReactNative\Microsoft.ReactNative.xml"   target="lib\uap10.0"/> 
     -->
+    <file src="$nugetroot$\natvis\Folly.natvis" target ="natvis\Folly.natvis"/>
```

Comment:
><file src="$nugetroot$\natvis\Folly.natvis" target ="natvis\Folly.natvis"/> [](start = 4, length = 75)

Our Nuget package distribution is only for binary ABI-safe API. This API does not use Folly and thus we must not need the Folly.natvis in it.

Reply:
I'm trying to include folly.natvis into msodll_40ui so it can be used when debugging folly dynamic objects in netui code. This package seems to be the source of the natvis file. Do you have any other recommendations on what msodll_40ui can use as the source of the natvis file?

Reply:
Should I perhaps create a new nuget just for the distribution of the natvis?

Reply:
we should just push the folly natvis to the folly repo

Reply:
If it was in the folly repo, whats a good way for it to be ingested and made available to the devmain build of msodll_40ui?

Reply:
not sure what devmain/msodll_40ui is (is that an office thing?) but you would just download the file in your build from the github repo like we do in RNW with the whole folly repo as a zip

Reply:
Yeah its is an office dll. I think we only support nuget and universal packages, not just just a loose zip file (At least I've never seen this). So we'd have to have some CI loop package and publish it into a nuget package anyway for consumption in the main office build process. I think we'd also like to have everything go through a package manager for proper versioning.

Reply:
I think we'd want to publish this as part of the desktop NuGet package instead of the experimental MSRN one. Devmain consumes the version of Folly compiled into the package in this repo, but as is pointed out, we're wanting to remove Folly from the boundary we expose at all. This is longer out, but in the future ViewManagers, NMs, etc, in devmain will be using the JSValue stuff Vlad wrote under the hood across the ABI boundary instead of Folly.

Reply:
Looks like you're copying to the right package now.